<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HikControl</class>
 <widget class="QWidget" name="HikControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="FOCUS_FAR_BTN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img/resources.qrc">
     <normaloff>:/icons/operations/operations/operations_2.png</normaloff>:/icons/operations/operations/operations_2.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FOCUS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>FOCUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="ZOOM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ZOOM</string>
   </property>
  </widget>
  <widget class="QLabel" name="FOCUS_STEP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>STEP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP_EDIT">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.30</string>
   </property>
  </widget>
  <widget class="QLabel" name="FRAME_RATE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>26</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SPEED_EDIT">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>95</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICE_IP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FPS_OPTION">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ICR_CHK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IR-CUT</string>
   </property>
  </widget>
  <widget class="QSlider" name="SPEED_SLIDER">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#SPEED_SLIDER_1::handle:horizontal{
	width: 6px;
	height: 8px;
	margin-top: -2px;
	margin-bottom: -2px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="FOCUS_NEAR_BTN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img/resources.qrc">
     <normaloff>:/icons/operations/operations/operations_3.png</normaloff>:/icons/operations/operations/operations_3.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ZOOM_OUT_BTN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img/resources.qrc">
     <normaloff>:/icons/operations/operations/operations_0.png</normaloff>:/icons/operations/operations/operations_0.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ZOOM_IN_BTN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img/resources.qrc">
     <normaloff>:/icons/operations/operations/operations_1.png</normaloff>:/icons/operations/operations/operations_1.png</iconset>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CONNECT_BTN">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img/resources.qrc">
     <normaloff>:/icons/operations/operations/connect.png</normaloff>:/icons/operations/operations/connect.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img/resources.qrc"/>
  <include location="img/resources.qrc"/>
  <include location="img/resources.qrc"/>
 </resources>
 <connections/>
</ui>
